skimage/future/__init__.py

```diff
@@ -1,8 +1,8 @@
-"""Functionality with an experimental API. Although you can count on the
-functions in this package being around in the future, the API may change with
-any version update **and will not follow the skimage two-version deprecation
-path**. Therefore, use the functions herein with care, and do not use them in
-production code that will depend on updated skimage versions.
+"""Functionality with an experimental API.
+
+.. warning::
+Although you can count on the functions in this package being around in the future, the API may change with any version update **and will not follow the skimage two-version deprecation
+path**. Therefore, use the functions herein with care, and do not use them in production code that will depend on updated skimage versions.
```

Comment:
That looks almost correct. Though it doesn't render properly as a warning yet as shown [on this site](https://output.circle-artifacts.com/output/job/da1a722b-49a7-4a1f-8773-f10b9873c933/artifacts/0/doc/build/html/api/skimage.future.html). You need to add an indent to the text block following the `warning::` bit. It should look similar to this https://github.com/scikit-image/scikit-image/blob/44da98e0a61f9382b0d96ad3973a1b83b2db1b5f/skimage/morphology/_queue_with_history.pxi#L2-L7

You can find these links by finding the list of automatic checks at the bottom of this pull request, and then clicking on "Details" in the "Check the rendered docs here!" job.

Our continuous integration system (CI) also throws a [warning](https://app.circleci.com/pipelines/github/scikit-image/scikit-image/5145/workflows/0a6efa6a-2dd3-4bae-b5fc-61b04e349a83/jobs/5195?invite=true#step-111-555) while building the HTML documentation.

Reply:
Did you try to build the documentation locally on your computer? This might be a good opportunity to try. You can find the [guide to building the docs here](https://scikit-image.org/docs/dev/contribute.html#building-docs). :)

Reply:
I tried building the docs using `dev.py docs` but it's throwing me this error `module 'devpy.cmds.util' has no attribute 'get_site_packages'; aborting.` I am using a macOS. Seems like the util module no longer has this function. I checked the docs.

Reply:
This is hopefully fixed now in https://github.com/scikit-image/scikit-image/pull/6789. You could try updating your `main` branch, merge it into this pull request branch and see if it works now. :)